File: GameShop.DAL/Entities/Publisher.cs
using System.Collections.Generic;

namespace GameShop.DAL.Entities
{
    public class Publisher : BaseEntity
    {
        public string CompanyName { get; set; }

        public string Description { get; set; }

        public string HomePage { get; set; }

        public ICollection<Game> Games { get; set; } = new List<Game>();
    }
}

File: AngularClient/src/app/core/models/Publisher.ts
import {ResourceModel} from "./resource-model";
import {Game} from "./Game";

export class Publisher extends ResourceModel<Publisher>
{
  CompanyName? : string;
  Description? : string;
  HomePage? : string;
  GameReadListDTOs? : Game[];

  constructor(model?: Partial<Publisher>) {
    super(model);
  }
}
